Source/DafnyCore/Compilers/CSharp/CsharpBackend.cs

```diff
@@ -48,11 +48,6 @@ public class CsharpBackend : ExecutableBackend {
     compilation = compilation.WithOptions(compilation.Options.WithOutputKind(callToMain != null ? OutputKind.ConsoleApplication : OutputKind.DynamicallyLinkedLibrary));
 
     var tempCompilationResult = new CSharpCompilationResult();
-    var libPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
Why are we no longer emitting the runtime dll?

Reply:
It's not being done on master either. Having these lines caused a crash.